### ISSUE TITLE
qtpass do not strip comments of gpg_id file

### ISSUE THREAD

**Issue by shemeshg** [IJHack/QtPass#625]
**Describe the bug**

in `pass` we have (line 101)
```
	local gpg_id
	while read -r gpg_id; do
		gpg_id="${gpg_id%%#*}" # strip comment
		[[ -n $gpg_id ]] || continue
		GPG_RECIPIENT_ARGS+=( "-r" "$gpg_id" )
		GPG_RECIPIENTS+=( "$gpg_id" )
	done < "$current"
```

however QtPass consider the complete line as UserId and no whatever before the ' # ' only 

**To Reproduce**
Create gpg_id entry in the .gpg_id file with remarks
```
21347213469hdsaklfha # username <username@whatever.com>
```

**Expected behavior**
QtPass should not care if entry has ramarks or not.

**Desktop (please complete the following information):**
- OS/Distribution: [ Ubuntu 20.04]
- Latest from git pull

- macosx
- Latest from git pull

probably fine tune of issue #452

**Comment by annejan** [comment]
Fxied in 1.4.0

**Comment by shemeshg** [comment]
Still not fixed therefore created PR #658,
Tested and with the PR, the users with comment in the `gpg_id` are now recognized.
